src/lib/helpers/downloadFile.ts

```diff
@@ -68,8 +68,8 @@ export const downloadFile = async (
   }
 
   const arrayBuffer = await res.arrayBuffer();
-  await fsPromises.mkdir(dirname(path), { recursive: true });
-  await fsPromises.writeFile(path, new Uint8Array(arrayBuffer));
+  await mkdir(dirname(path), { recursive: true });
```

Comment:
what is `path`? I'd name it something like `outputPath`, so meaning is clear

what will happen is that directory already exists, will it fail or continue?

Reply:
`outputPath` would be better. directory that exists results in a rejection only when `recursive` is `false`.